iOS Example/iOS Example/AppDelegate.swift

```diff
@@ -5,15 +5,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-    
+    func applicationDidFinishLaunching(_ application: UIApplication) {
+        window = UIWindow(frame: UIScreen.main.bounds)
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-        window = UIWindow(frame: UIScreen.mainScreen().bounds)
-        
         window?.rootViewController = ViewController()
         window?.makeKeyAndVisible()
-        window?.backgroundColor = UIColor.whiteColor()
-        
-        return true
```

Comment:
I'm not sure why it's considered a remove. I replaced it with ` = .white`. 
Also `applicationDidFinishLaunching(_:)` doesn't require a return value anymore.